core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -229,6 +229,7 @@ private[spark] class TaskSetManager(
       index: Int,
       resolveRacks: Boolean = true,
       speculatable: Boolean = false): Unit = {
+    if (isZombie) return
```

Comment:
A zombie TSM may call this while handling failed task.

Reply:
can you put it as code comment?